test/functional/counter-test.js

```diff
@@ -0,0 +1,63 @@
+import { assert } from 'chai';
+import sinon from 'sinon';
+import { Dispatcher } from '../../.dist/index';
+import CounterStore from './counter-store';
+
+describe('Counter', () => {
+
+    let spyA = sinon.spy();
+    let spyB = sinon.spy();
```

Comment:
So that I can test `"should remove a change listener"`. I remove B, and make sure that it calls A and not B.